lib/iris/tests/unit/data_manager/test_DataManager.py

```diff
@@ -158,6 +190,13 @@ def test_lazy(self):
         expected = "{}({!r})".format(self.name, self.lazy_array)
         self.assertEqual(result, expected)
 
+    def test_dataless(self):
```

Comment:
@ESadek-MO Rename to describe better what it's testing ... `test_dataless_repr` ?